### PR TITLE
Catalog with redis

### PR DESCRIPTION
**Pull request by inessaatntt**

### CODE REVIEW

File: src/Services/Catalog/Catalog.API/Extensions/RedisCacheExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+using Microsoft.Extensions.Caching.Distributed;
+
+namespace Catalog.API.Extensions
+{
+    public static class RedisCacheExtensions
+    {
+        private static IDistributedCache distributedCache;
+        private static IDistributedCache DistributedCache
+        {
+            get { return distributedCache; }
```
Comment (esmitsnttdata) [inline src/Services/Catalog/Catalog.API/Extensions/RedisCacheExtensions.cs]: I don't see this private variable used inside of the class. Does it have an alternative purpose?

File: src/Services/Catalog/Catalog.API/Model/CachedCatalogService.cs [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+                await _cache.TrySetAsync(filter.Key, item);
+            }
+            return item;
+        }
+
+        public async Task<IEnumerable<CatalogItem>> ListAsync(ISpecification<CatalogItem> spec, CacheCatalogFilter filter)
+        {
+            if (filter == null)
+                return null;
+
```
Comment (esmitsnttdata) [inline src/Services/Catalog/Catalog.API/Model/CachedCatalogService.cs]: I would have created 3 methods , 2 empty methods that call a full private ListAsync(ISpecification<CatalogItem> spec, CacheCatalogFilter filter,string ids = "") and have that method consider if the Ids  string is blank or not.

File: src/Services/Catalog/Catalog.API/Extensions/RedisCacheExtensions.cs (hunk 1 above)
Comment (inessaatntt) [reply src/Services/Catalog/Catalog.API/Extensions/RedisCacheExtensions.cs]: indeed a property is enough.

File: src/Services/Catalog/Catalog.API/Model/CachedCatalogService.cs (hunk 2 above)
Comment (inessaatntt) [reply src/Services/Catalog/Catalog.API/Model/CachedCatalogService.cs]: sure, thanks!

File: src/Services/Catalog/Catalog.UnitTests/Application/CatalogControllerTest.cs [hunk 3]
```diff
@@ -20,6 +20,7 @@ public CatalogControllerTest()
             _catalogServiceMock = new Mock<ICatalogService>();
         }
 
+        /*
         [Fact]
         public async Task Get_catalog_items_success()
```
Comment (esmitsnttdata) [inline src/Services/Catalog/Catalog.UnitTests/Application/CatalogControllerTest.cs]: Please don't leave commented code.

Comment (inessaatntt) [reply src/Services/Catalog/Catalog.UnitTests/Application/CatalogControllerTest.cs]: no changes on test projects, please ignore while doing the review.

**Pull request closed without merging**